mm2src/lp_swap/maker_swap.rs

```diff
@@ -118,11 +119,11 @@ pub struct MakerSwapData {
 pub struct MakerSwapMut {
     data: MakerSwapData,
```

Comment:
Could you please add `maker_coin_swap_contract_addr` and `taker_coin_swap_contract_addr` to MakerSwapData and TakerSwapData? We once discussed that swap contract address can potentially change on MM2 restarts which can result in MM2 looking for swap payments at wrong smart contract which may in turn result into double payment send and maybe other very unwanted consequences.

Reply:
Should I implement the swap contract checking and switching on device restart during a swap? It will take a long time. Or we will implement it on next iteration with Native mode?

Reply:
Yeah, I think it's fine to implement it in next iteration along with QRC20 native mode support to not delay this PR merging.